resources/iam-policies.go

```diff
@@ -74,6 +84,16 @@ func (e *IAMPolicy) Remove() error {
 	return nil
 }
 
+func (policy *IAMPolicy) Properties() types.Properties {
+	properties := types.NewProperties()
+
+	properties.Set("Name", policy.name)
+	properties.Set("ARN", policy.arn)
+	properties.Set("Path", policy.path)
+	properties.Set("PolicyID", policy.policyId)
+	return properties
+}
+
 func (e *IAMPolicy) String() string {
-	return e.arn
+	return e.name
```

Comment:
You cannot change the `String` output, as this would break existing aws-nuke configs.